linera-service/src/storage.rs

```diff
@@ -36,6 +36,8 @@ use {
     tracing::debug,
 };
 
+const DEFAULT_NAMESPACE: &'static str = "table_linera";
```

Comment:
(Ah, my bad. The `'static` is implicit.)

Reply:
Yes, it still works. Though clippy should have detected it.